forc/src/ops/forc_init.rs

```diff
@@ -97,7 +35,38 @@ fn print_welcome_message() {
 }
 
 pub fn init(command: InitCommand) -> Result<()> {
-    let project_name = command.project_name;
+    let project_dir = match &command.path {
+        Some(p) => PathBuf::from(p),
+        None => {
+            std::env::current_dir().context("Failed to get current directory for forc init.")?
+        }
+    };
+
+    if !project_dir.is_dir() {
+        anyhow::bail!("'{}' is not a valid directory.", project_dir.display());
+    }
+
+    if project_dir.join("Forc.toml").exists() {
```

Comment:
Use the constant `constants::MANIFEST_FILE_NAME` here and below instead of `"Forc.toml"`.